src/main/java/com/google/step/youtube/YoutubeInfoScraper.java

```diff
@@ -1,78 +1,95 @@
 
 package com.google.step.youtube;
 
-import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
-import com.google.api.client.http.javanet.NetHttpTransport;
-import com.google.api.client.json.jackson2.JacksonFactory;
-import com.google.api.services.youtube.YouTube;
-import com.google.api.services.youtube.model.ChannelListResponse;
-import com.google.api.services.youtube.YouTubeRequestInitializer;
 import com.google.api.services.youtube.model.Channel;
-import com.google.api.services.youtube.model.ChannelContentDetails;
-import com.google.api.services.youtube.model.ChannelContentDetails.RelatedPlaylists;
-import com.google.api.services.youtube.model.PlaylistItemListResponse;
+import com.google.api.services.youtube.model.ChannelListResponse;
 import com.google.api.services.youtube.model.PlaylistItem;
+import com.google.api.services.youtube.model.PlaylistItemSnippet;
+import com.google.api.services.youtube.model.PlaylistItemListResponse;
+import com.google.api.services.youtube.YouTube;
+import com.google.api.services.youtube.YouTubeRequestInitializer;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Date;
 import java.util.List;
-import java.io.IOException; 
 import java.util.Optional;
 
 /**
-*  Scrapes a channel's upload playlist and scrapes the channel's videos + descriptions 
-*/
+ * Scrapes a channel's upload playlist and scrapes the channel's videos +
+ * descriptions
+ */
 public class YoutubeInfoScraper {
 
-  // TODO: Add seperate file to hold API Key  
-  private static final String API_KEY = "";
-  private static final String APPLICATION_NAME = "promotube";
+    // TODO: Add seperate file to hold API Key
+    private static final String API_KEY = "";
+    private static final String APPLICATION_NAME = "promotube";
 
-  private final YouTube youTubeClient;
+    private final YouTube youTubeClient;
 
-  public YoutubeInfoScraper(YouTube youTubeClient) {
-      this.youTubeClient = youTubeClient;
-  }
+    public YoutubeInfoScraper(final YouTube youTubeClient) {
```

Comment:
As a note, for whether to put finals in method params/variables, we often leave it out since we always assume that it will never be reassigned (that is the best practice anyway), or else you'll want to add it to every param for consistency as you've done here. It's really just to reduce clutter on having to add final everywhere, but code immutability is generally good too. That's a choice and not enforced in the style guide so up to you if you want to keep the finals, but it's it's done out of complaints by the IDE then feel free to keep it like this.

Reply:
Removed finals.  There was an option in VScode to add finals to all variables.